app/views/govuk_publishing_components/components/_table.html.erb

```diff
@@ -2,46 +2,63 @@
   caption ||= nil
   head ||= []
   rows ||= []
-  first_cell_is_header ||=false
+  first_cell_is_header ||= false
   caption_classes ||= nil
-  sortable ||=false
+  sortable ||= false
+  filterable ||= false
 %>
 
-<%= GovukPublishingComponents::AppHelpers::TableHelper.helper(self, caption, {
-    sortable: sortable,
-    caption_classes: caption_classes
-  }) do |t| %>
+<div <% if filterable %> data-module="table-filter" <% end %> >
+  <% if filterable %>
+    <div class="govuk-table-filter">
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Search"
```

Comment:
All component text should be added to and then pulled in from the translation files (`en.yml`) e.g.

```erb
<%= t("components.table.filter_label") %>
```

Same for the 'no results' text.

Reply:
Makes sense: the text has been added to the translation files (keys only in locales other than `en`)